rosa_architecture/rosa_policy_service_definition/rosa-oidc-overview.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="rosa-oidc-overview"]
+= OpenID Connect Overview
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-oidc-overview
+toc::[]
+
+OpenID Connect (OIDC) allows users to provide a single set of credentials and gain access to multiple applications. Every time a user signs into an application or service using OIDC, the user is redirected to the OIDC identity provider.
+
+The OIDC protocol uses a configuration URL that contains the necessary information to authenticate a user's identity. The protocol responds to the provider with the credentials needed for the provider to validate the user and sign them in.
+
+{product-title} clusters can use OIDC to verify the authenticity of users accessing the clusters.
```

Comment:
All the changes in this file may be misleading. Although OIDC supports everything it says here, that's not how ROSA uses it.

OIDC is used by in-cluster operators to authenticate against the AWS account in which the cluster is running, so that the operators can access cloud resources directly. There's never user interaction in this context.